docs/user/getting_started.rst

```diff
@@ -85,15 +85,9 @@ referred to as ``$WECSIM`` (e.g. ``C:/User/Documents/GitHub/WEC-Sim``).
 Step 1. Add WEC-Sim to MATLAB Path
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Open the ``$WECSIM/source/wecSimStartup.m`` file.
+Open the ``$WECSIM folder and run the "activate_wecsim.m" script.
```

Comment:
```suggestion
Open the ``$WECSIM`` folder and run the "activate_wecsim.m" script.
```

Reply:
@jtgrasb Thanks for the comments and the fixes. To answer your questions,
1. Yes, the user needs to run activate script when they start using wecsim each time. The advantage of this is that. it ensures that the source folder is correctly added to the path, and no other files with name shadows are. There is always the risk that the user has similar filenames such as cylinder or sphere for instance. using the activate/deactivate workflow will prevent file path issues,
2. The work folder is actually a file dump folder. The slrpj files and other supporting files created by Simulink are dumped there. 

So a typical workflow would be,
1. User activates wecsim. This action populates the path with the source code,
2. Now user is free to work anywhere else without affecting the wecsim folder.

Let me know if that makes sense

Reply:
Okay, this makes sense, thanks!

In that case, I would recommend including something in the documentation about running the activate_wecsim file each time wec-sim is started and the reason for the work folder as this is a change from the previous methods. Also, I think it would be confusing to still have the wecsimStartup file in the source folder so I would recommend getting rid of that unless we have a reason to keep it. 

Let me know if that all makes sense and any thoughts you have.

Reply:
1. I agree and updated the getting started doc accordingly.
2. @nathanmtom  I can do that but do you see any downside to removing that?

Reply:
@nathanmtom  
@jtgrasb marked this as resolved. I am not confident about removing the the wecsimStartup file..